core-bundle/contao/dca/tl_page.php

```diff
@@ -245,6 +237,24 @@ static function ($value) {
 			'eval'                    => array('title_tag_callback'=>array('tl_page', 'getTitleTag'), 'titleFields'=>array('pageTitle', 'title'), 'tl_class'=>'clr'),
 			'sql'                     => null
 		),
+		'robots' => array
+		(
+			'search'                  => true,
+			'inputType'               => 'select',
+			'options'                 => array('index,follow', 'index,nofollow', 'noindex,follow', 'noindex,nofollow'),
+			'eval'                    => array('tl_class'=>'w50'),
+			'sql'                     => "varchar(32) NOT NULL default ''"
+		),
```

Comment:
You should generally avoid unrelated changes. Please move the `robots` field back to its original position for easier review.